### ISSUE TITLE
add default files into derivatives datasets

### ISSUE THREAD

**Issue by Remi-Gau** [cpp-lln-lab/bidspm#754]
- gitignore for .DS_store and similar things
- README.md
- copy / update participants.tsv from raw

**Comment by monique2208** [comment]
Is there a specific reason the participants file should be copied to the derivatives. I am getting a warning about it, but I am hesitant to copy the data over into the derivatives. I am not adding any derived information to it and having the same information twice creates the possibility for inconsistent data.

**Comment by Remi-Gau** [comment]
hum I think you are probably right about this one: if the participant.tsv is not modified it makes sense to NOT copy it.

I think I did that to avoid the warning that BIDS matlab throws if your dataset does not contain a participant.tsv.

To you, would it make sense to then update bids matlab to only throw this warning for raw datasets and not derivatives?

**Comment by monique2208** [comment]
Yes, I think so. So that is required for the dataset, but not necessarily in the derivates.

**Comment by Remi-Gau** [comment]
cool! 

will PR on bids matlab first and then change bidspm to revert this extra copy

**Comment by Remi-Gau** [comment]
this should be fixed #1188
